src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProjectFactory.cs

```diff
@@ -106,7 +107,7 @@ public async Task<ProjectSystemProject> CreateAndAddToWorkspaceAsync(string proj
                     name: projectSystemName,
                     assemblyName,
                     language,
-                    compilationOutputFilePaths: default, // will be updated when command line is set
+                    compilationOutputFilePaths: new(creationInfo.IntermediateOutputPath),
```

Comment:
Can we rename this parameter of ProjectAttributes to compilationOutputInfo or something? This made it sound like this was an ImmutableArray and I was worried that we were creating an array with a potentially null element or something.

Reply:
That caused a few file changes :D